xsuite/src/proxy/proxy.ts

```diff
@@ -29,22 +29,38 @@ export class Proxy {
     this.blockNonce = params.blockNonce ?? null;
   }
 
-  fetchRaw(path: string, data?: any) {
-    const baseUrl = this.proxyUrl;
-    const init: RequestInit = { headers: this.headers };
-    const url = new URL(`${baseUrl}${path}`);
-    if (this.blockNonce !== null) {
-      url.searchParams.append("blockNonce", this.blockNonce.toString());
+  private makeUrl(
+    path: string | URL,
+    params: Record<string, { toString: () => string } | undefined | null> = {},
+    override: boolean = false,
```

Comment:
Is there a case where somebody would want `override` to be false?

Reply:
I don't have any case in mind

Reply:
Okay, so in this case, is there a reason to keep this argument?